src/fsharp/PatternMatchCompilation.fs

```diff
@@ -363,37 +363,37 @@ let rec removeActive x l =
 let getDiscrimOfPattern (g: TcGlobals) tpinst t = 
     match t with 
     | TPat_null _m -> 
-        Some(Test.IsNull)
+        Some(DecisionTreeTest.IsNull)
     | TPat_isinst (srcty,tgty,_,_m) -> 
-        Some(Test.IsInst (instType tpinst srcty,instType tpinst tgty))
+        Some(DecisionTreeTest.IsInst (instType tpinst srcty,instType tpinst tgty))
     | TPat_exnconstr(tcref,_,_m) -> 
-        Some(Test.IsInst (g.exn_ty,mkAppTy tcref []))
+        Some(DecisionTreeTest.IsInst (g.exn_ty,mkAppTy tcref []))
     | TPat_const (c,_m) -> 
-        Some(Test.Const c)
+        Some(DecisionTreeTest.Const c)
     | TPat_unioncase (c,tyargs',_,_m) -> 
-        Some(Test.UnionCase (c,instTypes tpinst tyargs'))
+        Some(DecisionTreeTest.UnionCase (c,instTypes tpinst tyargs'))
     | TPat_array (args,ty,_m) -> 
-        Some(Test.ArrayLength (args.Length,ty))
+        Some(DecisionTreeTest.ArrayLength (args.Length,ty))
     | TPat_query ((pexp,resTys,apatVrefOpt,idx,apinfo),_,_m) -> 
-        Some(Test.ActivePatternCase (pexp, instTypes tpinst resTys, apatVrefOpt, idx, apinfo))
+        Some(DecisionTreeTest.ActivePatternCase (pexp, instTypes tpinst resTys, apatVrefOpt, idx, apinfo))
     | _ -> None
 
 let constOfDiscrim discrim =
     match discrim with 
-    | Test.Const x -> x
+    | DecisionTreeTest.Const x -> x
     | _ -> failwith "not a const case"
 
 let constOfCase (c: DecisionTreeCase) = constOfDiscrim c.Discriminator
 
 /// Compute pattern identity
 let discrimsEq (g: TcGlobals) d1 d2 =
   match d1,d2 with 
-  | Test.UnionCase (c1,_),    Test.UnionCase(c2,_) -> g.unionCaseRefEq c1 c2
-  | Test.ArrayLength (n1,_),   Test.ArrayLength(n2,_) -> (n1=n2)
-  | Test.Const c1,              Test.Const c2 -> (c1=c2)
-  | Test.IsNull ,               Test.IsNull -> true
-  | Test.IsInst (srcty1,tgty1), Test.IsInst (srcty2,tgty2) -> typeEquiv g srcty1 srcty2 && typeEquiv g tgty1 tgty2
-  | Test.ActivePatternCase (_,_,vrefOpt1,n1,_),        Test.ActivePatternCase (_,_,vrefOpt2,n2,_) -> 
+  | DecisionTreeTest.UnionCase (c1,_),    DecisionTreeTest.UnionCase(c2,_) -> g.unionCaseRefEq c1 c2
```

Comment:
Stylistic, I'd propose to align the right hand expressions and the left tuple since the diff is going to show those lines have changed and it currently look a bit cramped.